WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -1198,6 +1203,11 @@ private void onCommentCreated(OnCommentChanged event) {
     }
 
     private void onCommentLiked(OnCommentChanged event) {
+        // send signal for listeners to perform any needed updates
+        if (mNote != null) {
```

Comment:
Same as above. Should we check if it's a success or not before firing a new event?